applications/DeepSpeed-Chat/training/step1_supervised_finetuning/main.py

```diff
@@ -62,11 +62,18 @@ def parse_args():
         'Where to store the data-related files such as shuffle index. This needs to be on a local storage of a node (not on a shared storage)'
     )
     parser.add_argument(
-        "--model_name_or_path",
+        "--model_name",
```

Comment:
why split name and path?

Reply:
Good question, AML components don't allow an argument to be either a string or a path as a path is a separate concept. So, I split it out to allow for either.

We could keep it as only one argument, but it would mean that when we want to use a HF model, we need to first download it and upload it to a blob in AML - a bit annoying.

Reply:
Done!